.github/workflows/windows.yml

```diff
@@ -1,10 +1,10 @@
 name: Windows builds
 on: push
 env:
-  MAX_WARNINGS_GCC_32bit_Debug:     243
-  MAX_WARNINGS_GCC_64bit_Debug:     359
-  MAX_WARNINGS_Clang_32bit_Debug:   105
-  MAX_WARNINGS_Clang_64bit_Debug:   160
+  MAX_WARNINGS_GCC_32bit_Debug:     234
+  MAX_WARNINGS_GCC_64bit_Debug:     350
+  MAX_WARNINGS_Clang_32bit_Debug:   100
+  MAX_WARNINGS_Clang_64bit_Debug:   155
```

Comment:
.. and I noticed the Coverity job checked in with 2 fewer issues too! Nice work @dreamer .